help/include/create-a-stream-instructions.md

```diff
@@ -0,0 +1,18 @@
+{start_tabs}
+
+{tab|desktop-web}
+
+{relative|stream|all}
+
+1. Click **Create stream** on the right, or click the **plus**
+   (<i class="fa fa-plus"></i>) icon in the upper right.
+
+1. Fill out the requested info, and click **Create** in the bottom right corner
+   of the menu.
```

Comment:
I feel like I would n't call this a menu. Form maybe?

Reply:
Right. I changed it to "form", but maybe "panel" would be better?

Reply:
Yeah, not sure. The title of the panel at this point is 'create stream', so maybe ...

> 1. Fill out the requested info, and click **Create** in the bottom right corner of the create stream panel.

That would kind of match us using 'stream settings panel' in other places.

Reply:
I think that makes sense. Done!